lib/modules/saveComments.js

```diff
@@ -39,10 +54,17 @@ const savedCommentStorage = Storage.wrapBlob('RESmodules.saveComments.savedComme
 |} => { throw new Error('Saved comment not found'); });
 let savedCommentIDs;
 
+
 module.beforeLoad = async () => {
 	savedCommentIDs = new Set(Object.keys(await savedCommentStorage.getAll()));
```

Comment:
Retrieving all comments replied to could become costly. One solution could be to first only retrieve the keys, and defer retrieving the data till actually needing to render the saved comment.

Reply:
I'm not sure how to fix this as it doesn't appear there is a method to return a `BlobWrapper` keys.

Reply:
Looks like the storage would have to be have to converted to `PrefixWrapper` in order to improve performance, but that can probably wait.